.github/CONTRIBUTING.md

```diff
@@ -58,28 +58,26 @@ pip install -e .
 
 #### Run Tests
 
-```
+```bash
 pytest tests
 ```
 
 #### Build Documentation Locally
 
-```
+```bash
 make docs
 ```
 
-
 #### Adding New Dependencies
 
-To add new dependencies to the project, make sure to alter the *environment.yml* file. Then to sync the dependencies from the *environment.yml* file to the *requirements-dev.txt* run the following command
+To add new dependencies to the project, make sure to alter the _environment.yml_ file. Then to sync the dependencies from the _environment.yml_ file to the _requirements-dev.txt_ run the following command
```

Comment:
this is fine, but `*this*` and `_this_` are equivalent, no?

Reply:
They are. This is for consistency with earlier in the document where underscores are used.
My motivation is to follow this rule provided by `markdownlint`:
https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md049.md

It's pedantry, ultimately.